src/lib/ui/Switch/Switch.tsx

```diff
@@ -0,0 +1,29 @@
+interface SwitchProps {
+  checked: boolean
+  onChange: (checked: boolean) => void
+  label?: boolean
+}
+
+function Switch({
+  checked,
+  onChange,
+  label = false,
+}: SwitchProps): JSX.Element {
+  return (
+    <div className='seam-switch-container'>
+      {label && (
+        <label className='seam-switch-label'>{checked ? 'On' : 'Off'}</label>
+      )}
+      <div
+        className={`seam-switch ${checked ? 'checked' : ''}`}
```

Comment:
See for example https://github.com/seamapi/react/blob/f1b539d44dad34390beeb3447ff2607e35d61ea2/src/lib/ui/Table/TableFilterBar/TableFilterItem.tsx#L19

More importantly, all class names must prefix with `seam-`

```suggestion
        className={classNames('seam-switch', { 'seam-switch-checked': checked })}
```

Reply:
https://github.com/seamapi/react/pull/377/commits/ea36ce2ba9d59df08e9b2616ac49878a31cfd1b6#diff-b3283d61cb3fff4ad3dd0313780301f04779102940031eadd5fd19e0e2e1c9efR20